### PR TITLE
add unvalidated deserialization

### LINKED ISSUE #177: Non-validating deserialize
I changed my mind while preparing for release. I still think the default should validate inputs, but we should offer an escape hatch that does not validate for deserialize heavy workloads from trusted sources.

### PR DESCRIPTION
**Pull request by saik0**
Closes #177

### CODE REVIEW

File: src/bitmap/serialization.rs [hunk 1]
```diff
@@ -118,7 +120,49 @@ impl RoaringBitmap {
...
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// use roaring::RoaringBitmap;
+    ///
+    /// let rb1: RoaringBitmap = (1..4).collect();
+    /// let mut bytes = vec![];
+    /// rb1.serialize_into(&mut bytes).unwrap();
+    /// let rb2 = RoaringBitmap::deserialize_from(&bytes[..]).unwrap();
```
Comment (Kerollmops) [inline src/bitmap/serialization.rs:142]:
```suggestion
    /// let rb2 = RoaringBitmap::deserialize_from_unvalidated(&bytes[..]).unwrap();
```

**Review by Kerollmops — CHANGES_REQUESTED** [review]
Thank you, I'll merge once this little fix is done.

File: src/bitmap/serialization.rs (hunk 1 above)
Comment (saik0) [reply src/bitmap/serialization.rs:142]: Good catch.

**Review by Kerollmops — CHANGES_REQUESTED** [review]
We can probably add this method to the `Treemap` type, too?

**Comment by saik0** [comment]
> We can probably add this method to the `Treemap` type, too?

Done.

**Pull request merged**